cylc/flow/scripts/config.py

```diff
@@ -102,6 +103,25 @@ def get_option_parser():
             "overrides any settings it shares with those higher up."),
         action="store_true", default=False, dest="print_hierarchy")
 
+    platform_listing_options_group = parser.add_option_group(
+        'Platform printing options:')
+    platform_listing_options_group.add_option(
+        '--platform-names',
+        help=(
+            'Print a list of platforms and platform group names from the '
+            'configuration.'
+        ),
+        action='store_true', default=False, dest='print_platform_names'
+    )
+    platform_listing_options_group.add_option(
+        '--platform-meta',
+        help=(
+            'Print a list of platforms and platform group names and metadata'
+            'from the configuration.'
+        ),
```

Comment:
I think we were originally considering only printing the metadata, but not it prints full platforms config so the command option name is wrong.

```suggestion
        '--platforms',
        help=(
            'Print platform and platform group configuration, including metadata.'
        ),
```

(And corresponding variable name changes in the code).

Reply:
Corresponding name changes in code still needed